repo-stats/simple_data.json

```diff
@@ -0,0 +1 @@
 for you.\nHome-page: https://github.com/joke2k/faker\nAuthor: joke2k\nAuthor-email: joke2k@gmail.com\nLicense: MIT License\nLocation: /edx/src/play/django_2_upgrade/blah/lib/python3.5/site-packages\nRequires: python-dateutil, text-unidecode, six\nRequired-by: factory-boy\nMetadata-Version: 2.1\nInstaller: pip\nClassifiers:\n  Development Status :: 5 - Production/Stable\n  Environment :: Console\n  Intended Audience :: Developers\n  Programming Language :: Python\n  Programming Language :: Python :: 2\n  Programming Language :: Python :: 2.7\n  Programming Language :: Python :: 3\n  Programming Language :: Python :: 3.4\n  Programming Language :: Python :: 3.5\n  Programming Language :: Python :: 3.6\n  Programming Language :: Python :: 3.7\n  Programming Language :: Python :: Implementation :: CPython\n  Programming Language :: Python :: Implementation :: PyPy\n  Topic :: Software Development :: Libraries :: Python Modules\n  Topic :: Software Development :: Testing\n  Topic :: Utilities\n  License :: OSI Approved :: MIT License\nEntry-points:\n  [console_scripts]\n  faker = faker.cli:execute_from_command_line\n"}, "edx-milestones": {"Name": "edx-milestones", "Version": "0.2.5", "details": "Name: edx-milestones\nVersion: 0.2.5\nSummary: Significant events module for Open edX\nHome-page: https://github.com/edx/edx-milestones\nAuthor: edX\nAuthor-email: None\nLicense: AGPL\nLocation: /edx/src/play/django_2_upgrade/blah/lib/python3.5/site-packages\nRequires: six, django-model-utils, Django, edx-opaque-keys\nRequired-by: \nMetadata-Version: 2.1\nInstaller: pip\nClassifiers:\n  Development Status :: 5 - Production/Stable\n  Environment :: Web Environment\n  Intended Audience :: Developers\n  License :: OSI Approved :: GNU Affero General Public License v3\n  Operating System :: OS Independent\n  Programming Language :: Python\n  Programming Language :: Python :: 2\n  Programming Language :: Python :: 2.7\n  Programming Language :: Python :: 3\n  Programming Language :: Python :: 3.6\n  Framework :: Django\n  Framework :: Django :: 1.11\nEntry-points:\n"}, "requests-oauthlib": {"Name": "requests-oauthlib", "Version": "1.1.0", "details": "Name: requests-oauthlib\nVersion: 1.1.0\nSummary: OAuthlib authentication support for Requests.\nHome-page: https://github.com/requests/requests-oauthlib\nAuthor: Kenneth Reitz\nAuthor-email: me@kennethreitz.com\nLicense: ISC\nLocation: /edx/src/play/django_2_upgrade/blah/lib/python3.5/site-packages\nRequires: requests, oauthlib\nRequired-by: social-auth-core\nMetadata-Version: 2.1\nInstaller: pip\nClassifiers:\n  Development Status :: 5 - Production/Stable\n  Intended Audience :: Developers\n  Natural Language :: English\n  License :: OSI Approved :: BSD License\n  Programming Language :: Python\n  Programming Language :: Python :: 2\n  Programming Language :: Python :: 2.7\n  Programming Language :: Python :: 3\n  Programming Language :: Python :: 3.4\n  Programming Language :: Python :: 3.5\n  Programming Language :
```

Comment:
Is this just here as sample output?

Reply:
Yep, I'll try adding tests soon and will make this smaller.